include/clang/Lex/ModuleMap.h

```diff
@@ -112,7 +113,15 @@ class ModuleMap {
   /// \brief The number of modules we have created in total.
   unsigned NumCreatedModules = 0;
 
+  /// In case a module has a export_as entry, it might have a pending link
+  /// name to be determined if that module is imported.
+  llvm::StringMap<llvm::StringSet<>> PendingLinkAsModule;
```

Comment:
I suggest using `TinyPtrVector<Module *>` rather than `StringSet`.

Reply:
The reason I used a StringSet is in the ASTReader answer below.